runtime/src/lib.rs

```diff
@@ -154,13 +180,21 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 42;
 }
 
+// Configure FRAME pallets to include in runtime.
+
 impl frame_system::Config for Runtime {
+	/// The basic call filter to use in dispatchable.
+	type BaseCallFilter = ();
+	/// Block & extrinsics weights: base values and limits.
+	type BlockWeights = RuntimeBlockWeights;
+	/// The maximum length of a block (in bytes).
+	type BlockLength = RuntimeBlockLength;
 	/// The identifier used to distinguish between accounts.
 	type AccountId = AccountId;
 	/// The aggregated dispatch type that is available for extrinsics.
 	type Call = Call;
 	/// The lookup mechanism to get account ID from whatever is passed in dispatchers.
-	type Lookup = IdentityLookup<AccountId>;
+	type Lookup = AccountIdLookup<AccountId, ()>;
```

Comment:
why not use AccountIndex as second parameter here.

Reply:
I am just asking since address is now MultiAddress, this only affects ocw. So I am not sure if the correction is right. And here we may also use Indices. I think you know about the system, may you give a brief idea about potential candidates?

Reply:
AccountIndex is fine, AccountIdLookup is simpler than introducing indices pallet.

Reply:
Currently seems we may omit the AccountIndex as second parameter here. When it's necessary, we may add it. Or you may provide why it is needed now?